README.md

```diff
@@ -28,6 +28,25 @@ To help triage or fix bugs, see the current [Ignition issues](https://github.com
 
 To validate a config for Ignition there are binaries for a cli tool called ignition-validate available [on the releases page][releases], and an online validator available [on the CoreOS website][online-validator].
 
+## Dracut
+
+For distributions that use dracut, there is an
+[ignition-dracut](https://github.com/coreos/ignition-dracut)
+repo which contains scripts and systemd units for boot-time
+execution. But it's very likely that distributions will have
+to do additional work in order to properly integrate with
+Ignition.
+
+## Branches
```

Comment:
There's already a section on this above, though I think this is clearer. Let's remove the above section.

Reply:
I took the opportunity to also name FCOS and RHCOS in that first paragraph!